crates/polars/tests/it/io/csv.rs

```diff
@@ -542,7 +542,21 @@ fn test_comment_lines() -> PolarsResult<()> {
     let file = Cursor::new(csv);
     let df = CsvReader::new(file)
         .has_header(false)
-        .with_comment_char(Some(b'#'))
+        .with_comment_char(Some("#"))
+        .finish()?;
+    assert_eq!(df.shape(), (3, 5));
+
+    let csv = r"1,2,3,4,5
+### this is a comment
```

Comment:
This would equally work with a single `#` char; could you update the example to something that Ould _only_ work with a multi-character comment prefix?